app/models/grda_warehouse/hud/client.rb

```diff
@@ -787,6 +787,27 @@ def self.disabled_client_ids
       disabled_client_scope.pluck(:id)
     end
 
+    def self.disabling_condition_client_scope
+      mre_t = Arel::Table.new(:most_recent_enrollments)
+      join = she_t.join(mre_t).on(she_t[:id].eq(mre_t[:current_id]))
```

Comment:
Does this join need to include DateDeleted is null?

Reply:
This is by id to the window table (`most_recent_enrollments`) so it shouldn't contain any deleted enrollments as long as they are excluded in the window.